### ISSUE TITLE
Move docs off notion into docusaurus or something?

### ISSUE THREAD

**Issue by endigma** [livebud/bud#36]
Notion is a dumpster fire, you have full access to github pages and a million other things, please setup something actually usable. Docusaurus, Jekyll, Hugo and a bunch of other SSGs will create docs (can even be searchable) from markdown source (which I'm p sure you can export from Notion).

**Comment by matthewmueller** [comment]
@endigma thanks for your comment. Your tone coming across text is a bit harsh, but I'm going to assume best intentions.

Notion was a temporary solution to be able to share Bud with everyone quicker. I think it makes sense for the documentation to be built with Bud, don't ya think? 😉

**Comment by endigma** [comment]
Unless you support markdown source, I'd suggest against it. Some people willing to contribute may not care about sveltekit or react or whatever else is supported, and markdown is more or less the universal language of docs at this point. In addition, readable as plain text.

**Comment by endigma** [comment]
In addition, a lot of pre-existing software exists that supports full text search and static site generation

**Comment by matthewmueller** [comment]
Docs will be in `/docs` as either markdown, mdx, or potentially https://markdoc.io/. They'll be pre-rendered to either HTML or Svelte before deploying.

**Comment by endigma** [comment]
Agolia is free for OSS, there's also a few go document search libraries

**Comment by stefanos82** [comment]
I came here to ask the same question, more or less.

To be honest with you, Notion behaves...weird, at least in my case.

I try to move the page up or down with my arrow keys, but would insist on following URL TOC type of thing and while I am in the middle of the page, it takes me straight on top which is so darn annoying (excuse my exasperation, but has happened thrice in a row :laughing: ).

It would be really convenient to have it inside `docs` as a markdown file or even as generated HTML for the sake of maintaining syntax highlighting styling.

**Comment by matthewmueller** [comment]
> I try to move the page up or down with my arrow keys, but would insist on following URL TOC type of thing and while I am in the middle of the page, it takes me straight on top which is so darn annoying (excuse my exasperation, but has happened thrice in a row 😆 ).

Yes agreed, it's really bad UX. I'll try and get this done sooner, rather than later!

**Comment by djboboch** [comment]
I recommend using https://squidfunk.github.io/mkdocs-material/ 

It is a theme for mkdocs that has many features build it, some include:

- Diagrams using mermaid.js
- Various formating
- Visual enhancements for better readability 

There is more in the [reference](https://squidfunk.github.io/mkdocs-material/reference/)

The package builds with one command and can be deployed easily to github pages with actions as per [docs](https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions).

**Comment by omar391** [comment]
There is another alternative: https://docsify.js.org/
- Themeable
- Searchable
- No static HTML compilation required
